diff --git a/share/scons/get_xmlrpc_config b/share/scons/get_xmlrpc_config
index d54325a..1cc6096 100755
--- a/share/scons/get_xmlrpc_config
+++ b/share/scons/get_xmlrpc_config
@@ -137,8 +137,8 @@ Configs=[
     # Configuration for fedora
     {
         :description => "pkg-config",
-        :client => exec_command("pkg-config xmlrpc_client++ xmlrpc++ --libs"),
-        :server => exec_command("pkg-config xmlrpc_server_abyss++ --static --libs")
+        :client => exec_command("pkg-config xmlrpc_client++ xmlrpc_util xmlrpc++ --libs"),
+        :server => exec_command("pkg-config xmlrpc_server_abyss++ xmlrpc++ xmlrpc_util --static --libs")
     },
     # Configuration using xmlrpc-c-config
     {
